### PR TITLE
fix: Add missing localization in efb

### LINKED ISSUE #7815: loc: Missing localization in 'SimBridge Remote' section on the EFB
### Aircraft Version

Experimental

### Description

It seems that in my PR https://github.com/flybywiresim/a32nx/pull/7708, I forgot to add localization. 

### References (optional)

_No response_

### Additional info (optional)

_No response_

### Discord Username (optional)

Alepouna🌙#9824

### PR DESCRIPTION
**Pull request by ImPlasmaTbh**
Fixes #7815 

## Summary of Changes

Updated the SimOptions page to now use loc for values for text under SimBridge connection related settings in EFB.
Updated en.json localization page to include keys that will be used as text in SimOptions setting in EFB

## Additional context

Discord username (if different from GitHub): implasma#0

## Testing instructions

Turn on FlyPad
Navigate to Settings, then Sim Options page
Scroll down till you get to SimBridge settings where you can see simbridge related settings.
Text should show properly in English (you should be able to see SimBridge Host Machine, This PC, Remote PC etc. in English properly)

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

File: .github/CHANGELOG.md [hunk 1]
```diff
@@ -77,6 +77,7 @@
 1. [LIGHTS] Movement of landing lights now requires power and position is output into LVAR -  @Maximilian-Reuter
 1. [CDU] Fix auto weight and balance import on INIT B during GSX boarding not using the target values - @Maximilian-Reuter
 2. [FAC] Improve sideslip estimation - @lukecologne (luke)
+1. [EFB] Added missing localization for SimBridge related settings in SimOptions page - @implasmatbh (Plasma)
```
Comment (tracernz) [inline .github/CHANGELOG.md:80]: A changelog is probably not strictly necessary as there's no user-visible change. PR looks good otherwise :+1:.

Comment (ImPlasmaTbh) [reply .github/CHANGELOG.md:80]: I see. Should I then remove it from the changelog?
Also I need to get a build of this plane to make sure that the changes actually work but GitHub is skipping the build test. Am I missing something to get this to work?

**Review by tracernz — APPROVED** [review]

**Comment by alepouna** [comment]
Make sure to notify #localization so keys are created on Localazy

**Comment by 2hwk** [comment]
Note: QA should probably download from artifact to ensure localazy keys are loaded.

**Pull request merged**